content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -277,6 +277,7 @@ different Kubernetes components.
 | `WatchBookmark` | `false` | Alpha | 1.15 | 1.15 |
 | `WatchBookmark` | `true` | Beta | 1.16 | 1.16 |
 | `WatchBookmark` | `true` | GA | 1.17 | - |
+| `WindowsGMSA` | `true` | GA | 1.18 | - |
```

Comment:
Was this feature previously alpha / beta? If so, list those stages as well.

Reply:
Done @sftim